components/application-configuration/ui/src/main/resources/PhenoTips/MeasurementsConfigurationEditor.xml

```diff
@@ -0,0 +1,386 @@
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </parse>
+      <use>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <multiSelect>0</multiSelect>
+        <name>use</name>
+        <number>3</number>
+        <prettyName>Use this extension</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators>|, </separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>currentPage|onDemand|always</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </use>
+    </class>
+    <property>
+      <cache>long</cache>
+    </property>
+    <property>
+      <code>var PhenoTips = (function (PhenoTips) {
+  var widgets = PhenoTips.widgets = PhenoTips.widgets || {};
+  widgets.MeasurementsConfigurationEditor = Class.create({
+
+    _ENHANCED_TOOL_CSS_CLASS : 'initialized',
+    _NEW_TOOL_SELECTOR: ':not(.initialized)',
+    
+    initialize : function (element) {
+      if (!element) return;
+      this._element = element;
+      this._form = element.up('form');
+      this._enhanceExtensionTools();
+      this._enableFieldDragDrop = this._enableFieldDragDrop.bind(this);
+      this._syncDefaultValues = this._syncDefaultValues.bind(this);
+      this._syncDefaultValues();
+    },
+
+    _enhanceExtensionTools : function () {
+      this._silenceMoveButtons();
+      this._enableFieldDragDrop();
+    },
+
+    _silenceMoveButtons : function () {
+      this._element.select('a.move' + this._NEW_TOOL_SELECTOR).invoke('observe', 'click', function(event) {
+        event.stop();
+      });
+      this._element.select('a.move' + this._NEW_TOOL_SELECTOR).invoke('addClassName', this._ENHANCED_TOOL_CSS_CLASS);
+    },
+
+    _syncDefaultValues: function() {
+      this._element.select('.extension-maquette').each((function(el) {
+        el.down('input[name$=default]').disabled = !!!el.up('#default');
+      }).bind(this));
+    },
+
+    _enableFieldDragDrop : function () {
+       var _this = this;
+       var sections = this._element.select('.designer-panel .section-maquette .section-contents');
+       sections.each (function (section) {
+         Sortable.create(section, {
+           tag : 'div',
+           only: 'extension-maquette',
+           format: /^.*\.([^.]+)$/,
+           containment: sections,
+           dropOnEmpty: true,
+           constraint: false,
+           handle : 'move',
+           scroll: window,
+           onUpdate : function(container) {
+            _this._syncDefaultValues();
+           }
+         });
+       });
+    },
+  });
+  return PhenoTips;
+}(PhenoTips || {}));
+
+document.observe("xwiki:dom:loaded", function(){
+  window.wiz = new PhenoTips.widgets.MeasurementsConfigurationEditor($('measurements-configuration-editor'));
```

Comment:
This is left over from the [form designer code](https://github.com/phenotips/phenotips/blob/master/components/vocabularies/hpo/ui/src/main/resources/PhenoTips/FormDesigner.xml#L875) upon which this was based. I assume it was for testing/debug purposes, but I don't see any harm in leaving it in.